### ISSUE TITLE
Momentjs is in maintenace mode

### ISSUE THREAD

**Issue by npalm** [philips-labs/terraform-aws-github-runner#3435]
## Descrption
Momentjs is in [maintenance mode](https://momentjs.com/docs/#/-project-status/). Therefor we should check if we can stop using moment. If so remove it from the Lambda's.

**Comment by naorpeled** [comment]
Hey @npalm,
I'd like to take this if it's still relevant.

**Comment by npalm** [comment]
Yes still relevant, I assume this can solved with standard nodejs.

**Comment by npalm** [comment]
The library looks fine to me. I would first check if the functions used are today standard in NodeJS. If not we can move to date-fns or another alternative could be luxon.

**Comment by naorpeled** [comment]
> The library looks fine to me. I would first check if the functions used are today standard in NodeJS. If not we can move to date-fns or another alternative could be luxon.

Sounds good to me,
will try to get to working on this sometime this week.
Will keep you posted :)
